src/blocks/media-block/components/save.js

```diff
@@ -0,0 +1,77 @@
+import { Component } from "@wordpress/element";
+import cls from 'classnames';
+
+export class MediaBlockSaveComponent extends Component{
+
+    render () {
+
+        const { attributes } = this.props;
+
+        const isLinkAvailable = attributes.url !== undefined && attributes.url !== "";
+
+        let titleText = isLinkAvailable ?
+            <a href={ attributes.url } className={"c-block__title-link u-theme--link-hover--dark"}> {attributes.title} </a> :
+            attributes.title;
+
+        const styles = cls(
+            'c-media-block c-block l-grid-wrap l-grid-wrap--3-of-7',
```

Comment:
Replace `l-grid-wrap--3-of-7` with `l-grid-wrap--4-of-7`

Reply:
Ok. For the front side in front.js, do I need to replace l-grid-wrap--4-of-7 with l-grid-wrap--5-of-7?

Reply:
Are you sure that this code correct for "HideSidebar" checkbox?

`const contentStyles = cls(
            'c-block__content l-grid-item u-spacing u-color--gray u-border-left--black--at-large u-theme--border-color--darker--left',
            {'l-grid-item--s--3-col l-grid-item--m--2-col l-grid-item--l--2-col l-grid-item--xl--2-col': attributes.alignment === "left"},
            {'l-grid-item--s--5-col l-grid-item--m--4-col l-grid-item--l--3-col l-grid-item--xl--4-col l-grid-item--xxl--3-col': attributes.alignment === "center"}
        )`

Reply:
I'm not confident that the code will all correct. I'm sorry. This was a complicated one to review, and I was very sidetracked with other pressing project today. 

Send me your updates @YauheniKapliarchuk and I will test it again, including with the sidebar hidden.

Reply:
Thank you